### PR TITLE
WPF-880302 : Add code snippets example for Selecting item using SelectedItem and SelectedIndex in Accordion control

### PR DESCRIPTION
**Pull request by Priyanka-Vijayasankar**
Description:
Added the code snippets example for Selecting item using SelectedItem and SelectedIndex in Accordion control.

### CODE REVIEW

**Comment by Priyanka-Vijayasankar** [comment]
Rebuild

File: wpf/Accordion/Selecting-Items.md [hunk 1]
```diff
@@ -15,10 +15,160 @@ Items can be selected programmatically using the properties [SelectedIndex](http
...
 
+{% tabs %}
+
+{% highlight XAML %}
+
+ <Grid>
+     <StackPanel>
+     <layout:SfAccordion SelectedIndex="2" Width="500" Height="200">
+         <layout:SfAccordionItem Header="WPF" Content="Essential Studio for WPF"/>
+         <layout:SfAccordionItem Header="SilverLight" Content="Essential Studio for WPF"/>
```
Comment (Manivannan-E) [inline wpf/Accordion/Selecting-Items.md:26]: @Priyanka-Vijayasankar Change the description content as we discussed and also remove the unwanted codes.

File: wpf/Accordion/Selecting-Items.md [hunk 2]
```diff
@@ -15,10 +15,160 @@ Items can be selected programmatically using the properties [SelectedIndex](http
...
+accordion.Items.Add(windowsPhoneItem);
+accordion.Items.Add(universalItem);
+accordion.SelectedIndex = 2;
+this.Content = accordion;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![WPF Accordion Item with SelectedIndex](Selecting-Items-images/wpf-accordion-selected-item-selectedindex.png)
```
Comment (Manivannan-E) [inline wpf/Accordion/Selecting-Items.md:87]: @Priyanka-Vijayasankar Change image alt name as **WPF Accordion item selection using SelectedIndex** image name as **wpf-accordion-selected-index-selection**

File: wpf/Accordion/Selecting-Items.md [hunk 3]
```diff
@@ -15,10 +15,160 @@ Items can be selected programmatically using the properties [SelectedIndex](http
...
+  {
+      public string Name { get; set; }
+      public string Description { get; set; }
+  }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![WPF Accordion Item with SelectedItem](Selecting-Items-images/wpf-accordion-selected-item-selecteditem.png)
```
Comment (Manivannan-E) [inline wpf/Accordion/Selecting-Items.md:170]: @Priyanka-Vijayasankar Change image alt name as **WPF Accordion item selection using SelectedItem** image name as **wpf-accordion-selected-item-selection**

File: wpf/Accordion/Selecting-Items.md (hunk 1 above)
Comment (Priyanka-Vijayasankar) [reply wpf/Accordion/Selecting-Items.md]: Update the mentioned scenario.

File: wpf/Accordion/Selecting-Items.md (hunk 2 above)
Comment (Priyanka-Vijayasankar) [reply wpf/Accordion/Selecting-Items.md]: Update the mentioned scenario.

File: wpf/Accordion/Selecting-Items.md (hunk 3 above)
Comment (Priyanka-Vijayasankar) [reply wpf/Accordion/Selecting-Items.md]: Update the mentioned scenario.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

**Pull request merged**